### PR TITLE
add event

### PR DESCRIPTION
**Pull request by tmoolmuang**
please check, thanks

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    @app = App.find(params[:app_id])
+    @event = Event.new
+  end
+
+  def create
+    @app = App.find(params[:app_id])
+    @event = Event.new
+    @event.name = params[:event][:name]
+    @event.app = @app
+
```
Comment (aspencer8111) [inline app/controllers/events_controller.rb]: Could probably clean this up a bit:

```
@app = App.find(params[:app_id])
@event = @app.events.new(name: params[:event][:name])
```

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+  get 'posts/new'
+
+  get 'posts/create'
+
+  get 'posts/edit'
+
+  get 'posts/update'
+
+  get 'posts/destroy'
+
```
Comment (aspencer8111) [inline config/routes.rb]: Why all of the named routes above? Why not `resources :posts`? Also are these routes needed?

File: spec/features/event_spec.rb [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      click_on 'Save'
+      expect(page).to have_content "Event was updated successfully"
+    end
+
+    scenario 'allows admin to edit any event' do
+      login_as(std_user)
+      create_app
+      create_event
+      click_on 'Sign Out'
+      login_as(admin_user)
```
Comment (aspencer8111) [inline spec/features/event_spec.rb]: We aren't testing anything as a user here. Maybe break this into its own test where a user signs in and then tries to edit an event, but sees a page with "you aren't authorized"?

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (tmoolmuang) [inline app/controllers/events_controller.rb]: fixed

File: config/routes.rb (hunk 2 above)
Comment (tmoolmuang) [inline config/routes.rb]: I did some tests and forgot to take them out.

File: spec/features/event_spec.rb (hunk 3 above)
Comment (tmoolmuang) [inline spec/features/event_spec.rb]: Fixed. And add few more test scenarios for app/event owner and non-owners.

**Comment by tmoolmuang** [comment]
done

**Pull request merged**